tests/unit/cli/test_baseline.py

```diff
@@ -83,7 +83,7 @@ def test_bandit_baseline(self):
         git_repo.index.commit('Initial commit')
         os.chdir(repo_directory)
 
-        with open('bandit.yaml', 'wt') as fd:
+        with open('bandit.yml', 'wt') as fd:
```

Comment:
Why is this being changed?

Reply:
because thats the new default filename and it seems fitting that we should test "default" YAML filename as part of baseline tests
other tests already test of custom provided filename

I will likely change the default filename to `.bandit.yml` or `bandit.yaml` (in that add a test case with custom filename) because neither PyCQA or openstack is using `bandit.yml` - I just took that name from one of the comments from #318 discussion.